### DISCUSSION TITLE
Description of command args for handlers

### DISCUSSION THREAD

**Discussion by Timmythetim** [cmu-sei/GHOSTS#137] [Q&A] [answered]
I just had a question about the command args for the handlers. I know that for browser handlers there is stickiness and max depth, but I can’t find anywhere a description of command args that are allowed for other handlers, like Notepad, Word, etc. Is there any file that depicts allowed command args?

**Comment by Timmythetim** [comment]
I realize that I have mixed up handler args and command args. Is there any file that describes how command args and handler args work in the timeline?

**Comment by sei-dupdyke** [comment, marked as answer]
Hi, sorry for late response, was on PTO.

For every handler, I try to have a detailed sample [available here](https://github.com/cmu-sei/GHOSTS/tree/master/src/Ghosts.Client/Sample%20Timelines).

Some further details are here on [this wiki page](https://github.com/cmu-sei/GHOSTS/wiki/Configuring-the-Windows-Client).

If you have questions on a specific one, I'm happy to answer those here.
